src/legacy/core_plugins/interpreter/public/functions/esaggs.js

```diff
@@ -79,9 +89,9 @@ export const esaggs = () => ({
     const response = await courierRequestHandler({
       searchSource: searchSource,
       aggs: aggs,
-      timeRange: get(context, 'timeRange', null),
-      query: get(context, 'query', null),
-      filters: get(context, 'filters', null),
+      timeRange: get(context, 'timeRange', { from: 'now-1M', to: 'now' }),
```

Comment:
Do you expect we're going to use the standard filter bar in the visual builder? This seems like a temporary fix that might be hard to remember, maybe you can make it more obvious that this is overriding those settings?

Reply:
> Do you expect we're going to use the standard filter bar in the visual builder?

Yes, I think this makes the most sense. And even if there will be another filter bar, it will have to provide the time range in some way.

This is the default value, it's not overriding anything if the timeRange is actually provided (which we will add when adding a filter bar). I actually have on my TODO list to bring these default values back into master because the current `null` default values actually break courier. In this separate PR I will discuss with app arch which default values actually make sense and reflect the result back into our branch. I added a TODO to find this easier in the futures so it won't be overlooked.